integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -279,6 +280,7 @@ public final void before() throws Exception {
     mFsMaster =
         mLocalTachyonClusterResource.get().getMaster().getInternalMaster().getFileSystemMaster();
     mMasterTachyonConf = mLocalTachyonClusterResource.get().getMasterTachyonConf();
+    TTLBucket.setTTlIntervalMs(mMasterTachyonConf.getLong(Constants.MASTER_TTLCHECKER_INTERVAL_MS));
```

Comment:
this is unrelated, but could you change the name of setTTlIntervalMs to setTTLIntervalMs? (or add a TODO / create a ticket)